core/src/block.rs

```diff
@@ -482,14 +482,14 @@ impl VersionedCandidateBlock {
     /// # Errors
     /// Forward errors from [`CandidateBlock::revalidate`]
     #[inline]
-    pub fn revalidate<const TX_ORIGIN: TransactionOrigin>(
+    pub fn revalidate<const IS_GENESIS: bool>(
```

Comment:
```suggestion
    pub fn revalidate<const IS_GENESIS: bool = false>(
```

I think this might be supported in the future which would make API much nicer for the default case

Reply:
this feature would be really nice to have